frontend/src/api/SearchPapers.jsx

```
@@ -167,15 +186,36 @@ const SearchPapers = () => {
               <button
-                onClick={() =>{
-                  console.log("paper:", paper)
-                  downloadPaper(paper.id, paper.original_filename)}}
-                className="mt-3 inline-flex items-center gap-2 text-sm bg-green-600 text-white px-3 py-1.5 rounded hover:bg-green-700 transition"
+                onClick={() => {
+                  console.log("paper:", paper);
+                  downloadPaper(paper.id, paper.original_filename);
+                }}
+                className="inline-flex items-center gap-2 text-sm bg-green-600 text-white px-3 py-1.5 rounded hover:bg-green-700 transition"
               >
                 <Download className="w-4 h-4" />
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)